lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -87,4 +95,6 @@ class DOM {
 
 if (typeof module !== 'undefined' && module.exports) {
   module.exports = DOM;
+} else if (self) {
```

Comment:
`typeof self !== 'undefined'` for all of these? I'm ambivalent between that and just having `else { }` since I can't come up with a case where module *and* self are both undefined :)